### PR TITLE
Login form page

### PR DESCRIPTION
**Pull request by AndiAJ**
New login form test page.

Used the page created on my repo https://andiaj.github.io/testapp/loginForm and works properly in the newly created "Saved logins in PWA" UI smoke test.

• The PWA is properly installed
• The login form works fine

### CODE REVIEW

**Review by irios3 — APPROVED** [review]
LGTM, thanks!

File: loginForm.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  <p class="header">Login Form</p>
+  <form>
+    <input id="username" type="text" placeholder="Username" /><br />
+    <input id="password" type="password" placeholder="Password" /><br />
+    <input type="submit" id="submit" />
+  </form>
+
+
+</html>
+
```
Comment (AaronMT) [inline loginForm.html:18]: Nit: Please add a title to the head and remove the blank space (line 16/17) (W3C HTML validator)

**Review by AaronMT — APPROVED** [review]
See nit

File: loginForm.html (hunk 1 above)
Comment (AndiAJ) [reply loginForm.html:18]: Done, thanks!

**Pull request merged**